src/pubsub/mod_pubsub_db.erl

```diff
@@ -124,7 +124,20 @@
 -callback remove_all_items(Nidx :: mod_pubsub:nodeIdx()) ->
     ok.
 
+-callback get_items(Nidx :: mod_pubsub:nodeIdx()) ->
+    {result, {[mod_pubsub:pubsubItem()], none}}.
```

Comment:
Breaks the convention of existing calls where return has format `{ok, Res}`.
Also even if main module expects some `none` value, IMO DB backend shouldn't worry about it and it should be responsibility of the node impl.

Reply:
It may break the convention but somehow it doesn't break the `node_call` framework.... I'll try to fix that with probably tons of refactored lines...